src/main/java/gil/server/controllers/PersonController.java

```diff
@@ -35,11 +35,12 @@ public PersonController(JSONDataStore dataStore) {
                     Integer pid = person.getId();
                     String name = person.getName();
                     String email = person.getEmail();
+                    String body = "{ \"id\": " + pid + ", \"name\": \"" + name + "\", \"email\": \"" + email + "\"}";
```

Comment:
This is basically, json, right? Can you use a json builder so it isn't string concatenation?

Reply:
Great suggestion. Thank you!